src/qt/pivx/topbar.cpp

```diff
@@ -608,8 +608,7 @@ void TopBar::updateTorIcon()
             ui->pushButtonTor->setChecked(true);
             ui->pushButtonTor->setButtonClassStyle("cssClass", "btn-check-tor", true);
         }
-        QString ip_port_q = QString::fromStdString(ip_port);
-        ui->pushButtonTor->setButtonText(tr("Tor Active: %1").arg(ip_port_q));
+        ui->pushButtonTor->setButtonText(tr("Tor Active"));
```

Comment:
could we maybe use `setToolTip()` to display the onion address? also, maybe a future PR could make this button an action button that allows for the onion address to be copied to the clipboard?

Reply:
sure, will add the tooltip.

I left a comment in the commit: ideally, a future good work would be to create a network-only information dialog, grouping and presenting the network/proxy related information properly (networks, addresses, reachability, if it's a limited net, etc). So we keep the topbar menu buttons as essentially actionables for sub-menus and/or other information screens/dialogs.

Still, atm, we know that no regular user will really use it. Something for the future when there is some free time to spend on extra things.